docs/notebooks/binning_slowly_changing_sources.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    _ = plt.hist(ens._source["midPointTai"].compute().tolist(), 1000)
</pre><p>All plots need to have labels, e.g., on first plot the x-axis should be either MJD, or time, or days, or midPointTai, and y-axis should be number of objects of N</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/lsstseries/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done.